runtime/tests/integration_tests.rs

```diff
@@ -101,6 +102,44 @@ fn test_locked_dv_slot() {
     model.step_until_output_contains("TEST EXCEPTION").unwrap();
 }
 
+#[test]
+// Check if the owner and vendor cert validty dates are present in RT Alias cert
+fn test_rt_cert_with_custom_dates() {
+    let mut opts = ImageOptions::default();
+
+    opts.vendor_config
+        .not_before
+        .copy_from_slice("20250101000000Z".as_bytes());
+
+    opts.vendor_config
+        .not_after
+        .copy_from_slice("20260101000000Z".as_bytes());
+
+    let mut own_config = opts.owner_config.unwrap();
+
+    own_config
+        .not_before
+        .copy_from_slice("20270101000000Z".as_bytes());
+    own_config
+        .not_after
+        .copy_from_slice("20280101000000Z".as_bytes());
```

Comment:
nit: Can we define these strings as variables/constants at the top of the function? That we don't have to edit in multiple places if they need to be changed.